pkg/workflows/sdk/workflow_spec.go

```diff
@@ -1,12 +1,129 @@
 package sdk
 
-import "github.com/smartcontractkit/chainlink-common/pkg/capabilities"
+import (
+	"cmp"
+	"iter"
+	"slices"
+	"strings"
+	"text/template"
```

Comment:
I'm a little worried about pulling in text/template to our WASM binaries.

Could we make helpers who live outside of the SDK to do this work?

Reply:
Sure, as long as everything we need is exported, then it doesn't matter where it lives.

Reply:
Moved to workflows package as a method on `DependencyGraph`